src/Microsoft.Diagnostics.Monitoring.WebApi/Controllers/DiagController.cs

```diff
@@ -203,17 +205,24 @@ public Task<ActionResult<IEnumerable<Models.ProcessIdentifier>>> GetProcesses()
             [FromQuery]
             Models.DumpType type = Models.DumpType.WithHeap,
             [FromQuery]
-            string egressProvider = null)
+            string egressProvider = null,
+            [FromQuery]
+            Models.PackageMode mode = Models.PackageMode.None)
```

Comment:
This needs model validation such that IncludeDacDbi must be used in conjunction with DiagSession.

Reply:
IncludeDacDbi implies DiagSession.